src/fastfield/serializer/mod.rs

```diff
@@ -84,13 +71,14 @@ impl CompositeFastFieldSerializer {
         Ok(())
     }
 
-    /// Start serializing a new [u8] fast field
-    pub fn new_bytes_fast_field_with_idx(
+    /// Start serializing a new [u8] fast field.
+    ///
+    /// The bytes will be stored as is, no compression will be applied.
+    pub fn new_bytes_fast_field(
```

Comment:
Nothing in the method name explains that it is return the thing that writes the value.